datasource.js

```diff
@@ -23,5 +23,12 @@ export class WalloraAPI extends RESTDataSource {
       }
     })
   }
-
+  async getCurrencySettings(v1AccessToken,v2AccessToken) {
+    return this.get("currencies",null, { 
+      headers: {
```

Comment:
now that the header is getting repeated in all calls, can you please extract the code to reuse